cluster/manifests/02-skipper-validation-webhook/deployment.yaml

```diff
@@ -51,11 +54,32 @@ spec:
         image: 926694233939.dkr.ecr.eu-central-1.amazonaws.com/production_namespace/teapot/skipper:v0.22.142
         args:
           - skipper
+          - -support-listener=:9981
           - --validation-webhook-enabled=true
           - --validation-webhook-address=:9085
           - --validation-webhook-cert-file=/etc/tls-certs/skipper-validation-webhook.pem
           - --validation-webhook-key-file=/etc/tls-certs/skipper-validation-webhook-key.pem
           - "--enable-advanced-validation={{ .Cluster.ConfigItems.enable_advanced_validation }}"
+          - "-metrics-flavour=prometheus"
```

Comment:
I assume the mixed hyphen count for the flags is fine?

Reply:
It does not matter if you have one or 2